src/main/kotlin/com/fantasy/ladbe/repository/custom/PlayerCustomRepository.kt

```diff
@@ -0,0 +1,14 @@
+package com.fantasy.ladbe.repository.custom
+
+import com.fantasy.ladbe.dto.PlayerDto
+import com.fantasy.ladbe.model.Player
+import org.springframework.data.domain.Page
+
+interface PlayerCustomRepository {
+
+    fun selectById(id: Long): Player?
+
+    fun selectAll(): List<Player>?
```

Comment:
list, map 을 리턴하는 경우 값이 없으면 null 대신 empty collection 을 리턴하도록 해야 이 함수를 호출하는 쪽에서 편해요:

```suggestion
    fun selectAll(): List<Player>
```

Reply:
```
fun getAllPlayers() =
        ResponseEntity(success(playerService.readAll()), HttpStatus.OK)


fun readAll() =
        playerRepository.selectAll().map { PlayerDto.Response.PlayerDetail().entityToDto(it) }
```

가이드 주신 위 부분과 함께 수정하겠습니다. null 관련하여 헤매고 있었는데 설명해주셔서 명확하게 이해된 것 같네요..👍